### ISSUE TITLE
sails.getBaseurl() returns a localhost URL when deployed (on Heroku)

### ISSUE THREAD

**Issue by fabianweb** [balderdashy/sails#2300]
I'm running Sails.js 0.10.5 and `sails.getBaseurl()` gives the URL `http://localhost:41241` although the app is deployed and running (on Heroku).

Any ideas how to solve this?

Thank you!

**Comment by loicsaintroch** [comment]
What about #2310 ?

**Comment by CWyrtzen** [comment]
Has any of this been resolved via work-around or subsequent release?

**Comment by sailsbot** [comment]
Thanks for posting, @fabianweb.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
